elements/pfe-cta/demo/index.html

```diff
@@ -113,20 +114,24 @@ <h1>Old school testing (custom backgrounds)</h1>
   <br />
   <div class="custom-light-band ">
     <h2>Custom light band with broadcast vars</h2>
-    <pfe-cta><a href="#">Default (color responsive)</a></pfe-cta>
-    <pfe-cta pfe-priority="primary"><a href="#">Primary: (color responsive)</a></pfe-cta>
-    <pfe-cta pfe-priority="secondary"><a href="#">Secondary: (color responsive)</a></pfe-cta>
+    <h3>Color responsive</h3>
+    <pfe-cta><a href="#">Default</a></pfe-cta>
+    <pfe-cta pfe-priority="primary"><a href="#">Primary</a></pfe-cta>
+    <pfe-cta pfe-priority="secondary"><a href="#">Secondary</a></pfe-cta>
+    <h3>Color override</h3>
     <pfe-cta pfe-priority="primary" pfe-color="accent"><a href="#">Primary accent</a></pfe-cta>
     <pfe-cta pfe-priority="secondary" pfe-color="accent"><a href="#">Secondary accent</a></pfe-cta>
     <pfe-cta pfe-priority="primary" pfe-color="complement"><a href="#">Primary complement</a></pfe-cta>
     <pfe-cta pfe-priority="secondary" pfe-color="complement"><a href="#">Secondary complement</a></pfe-cta>
     <pfe-cta pfe-priority="primary" pfe-color="lightest"><a href="#">Primary lightest</a></pfe-cta>
   </div>
   <div class="custom-dark-band">
-    <h2 style="color:white;">Custom dark band with broadcast vars</h2>
-    <pfe-cta><a href="#">Default (color responsive)</a></pfe-cta>
-    <pfe-cta pfe-priority="primary"><a href="#">Primary (color responsive)</a></pfe-cta>
-    <pfe-cta pfe-priority="secondary"><a href="#">Secondary (color responsive)</a></pfe-cta>
+    <h2 style="color:white;">Custom dark band with custom broadcast variables</h2>
+    <h3 style="color: white">Color responsive</h3>
+    <pfe-cta><a href="#">Default</a></pfe-cta>
+    <pfe-cta pfe-priority="primary"><a href="#">Primary</a></pfe-cta>
+    <pfe-cta pfe-priority="secondary"><a href="#">Secondary</a></pfe-cta>
+    <h3 style="color: white">Color override</h3>
```

Comment:
just moved the descriptions about color override vs. color responsive out of the buttons and into headings